1.9/networking/load-balancing-vips/virtual-ip-addresses.md

```diff
@@ -97,4 +96,10 @@ Here is a sample response:
 
 You can use this VIP to address any one of the Kafka brokers in the cluster.
 
+## FAQs
+
+### Connections seem to close at random times
+
+This is seen most often with applications that have long lived connections like databases (e.g. PostgreSQL). The solution may be to turn on keepalives. The keepalive can be an application specific mechanism like a heartbeat, or something in the protocol like a TCP keepalive. The keepalive is necessary because to a load balancer, an idle and a dead connection look the same (no packets), so they are treated as dead. The default timeout (depends on kernel configuration) is usually 5 minutes.
```

Comment:
- `This behavior is often experienced with applications that have long lived connections, such as databases (e.g. PostgreSQL).`
- `To fix, try turning on keepalives.`
- `A keepalive is required because a load balancer treats an idle and a dead connection the same.`
- `The default timeout depends on the kernel configuration, but is usually 5 minutes.`